server/couchDB/couchSeeder.js

```diff
@@ -0,0 +1,25 @@
+const createDB = require('./');
+const generateCouchData = require('./generateCouchData');
+const batchSize = 1000;
+
+(async () => {
+  const workspaces = await createDB();
+  let docs = [];
+  let batch = 0;
+  for(let i = 1; i <= 10000000; i++) {
+    const dataFormat = generateCouchData(i);
+    docs.push(dataFormat);
+    batch++;
+    if(batch === batchSize) {
```

Comment:
The final batch might not be pushed created, if it's not a multiple of the total records. E.g. 10000000 / 1000 = whole number. But 9999999 / 1000 is not and you'll be left with less than 10M records.